go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/docker/docker v20.10.24+incompatible
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/graph-gophers/graphql-go v1.5.0
+	github.com/hamba/avro/v2 v2.12.0
```

Comment:
any special reason you chose that specific package rather than the Linkedin package for example?

Reply:
No specific reasons other than hamba/avro is 2-3 times faster in encoding/decoding than linkedin package.
But i'm happy to swap hamba/avro with linkedin?
https://github.com/hamba/avro#benchmark